src/runner.go

```diff
@@ -65,7 +65,10 @@ func verifyYamlFile(yamlData []byte) bool {
 		log.Errorln(err)
 		return false
 	}
-	stdin.Close()
+
+	if err := stdin.Close(); err != nil {
+		log.Errorln("stdin was unexpectedly already closed: ", err)
```

Comment:
@r0x0d Not sure if we want to log it as Error or Info, I am torn here - I chose error as it may indicate some unexpected behavior (but at the same time if it was already closed and no other errors are there it shouldn't cause any additional problems)

Reply:
But if we have an error, then it means that the file was not closed correctly, right? In that specific case, I believe that logging as error is more appropriate here.

Reply:
depends, with closing file it is that either some IO operation was still in progress or close was already called, but it is closed anyway, with stdin an writer it isn't specified in documentation - os probably any error.

Lets leave error, it makes sense (I was just overthinking again)